### DISCUSSION TITLE
Remote input error

### DISCUSSION THREAD

**Discussion by gordon848** [andyholmes/valent#703] [Help]
I try to use remote input to control my laptop. But Iam getting this error:

`12:18:35.1295        valent-clipboard-plugin:    DEBUG: valent_clipboard_plugin_handle_clipboard_connect(): expected "content" field holding a string
12:18:38.5089               valent-xdp-input:  WARNING: on_session_created(): GDBus.Error:org.freedesktop.DBus.Error.InvalidArgs: Keine derartige Schnittstelle »org.freedesktop.portal.ScreenCast«
12:18:38.7138               valent-xdp-input:  WARNING: on_session_created(): GDBus.Error:org.freedesktop.DBus.Error.InvalidArgs: Keine derartige Schnittstelle »org.freedesktop.portal.ScreenCast«
12:18:38.9177               valent-xdp-input:  WARNING: on_session_created(): GDBus.Error:org.freedesktop.DBus.Error.InvalidArgs: Keine derartige Schnittstelle »org.freedesktop.portal.ScreenCast«
`
I use arch, valent-git (aur) and XFCE (x11). Is there a package or something missing? Sorry, I tried to find a solution online but wasn't successful.

**Comment by andyholmes** [comment]
You'll need a portal implementation that supports the Remote Desktop interface. xdg-desktop-portal-gnome and xdg-desktop-portal-kde support it, so I guess XFCE uses xdg-desktop-portal which may not have a default implementation.

I'm not sure if installing either of those will help, or if they require something in XFCE to work though, sorry :/

**Comment by gordon848** [comment]
I tried the packages you mentioned. Unfortunately without success. So, I guess like you already said, it needs something in XFCE to work.
